go/vt/vtgate/vstream_manager.go

```diff
@@ -604,7 +608,8 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 			Options:      options,
 		}
 		var vstreamCreatedOnce sync.Once
-		log.Infof("Starting to vstream from %s, with req %+v", topoproto.TabletAliasString(tablet.Alias), req)
+		tabletAliasString := topoproto.TabletAliasString(tablet.Alias)
+		log.Infof("Starting to vstream from %s, with req %+v", tabletAliasString, req)
```

Comment:
Any reason to return alias over hostname? Seems a lot of time the first step is done with the hostname (eg ssh in to get logs).

I know this mapping is in vtadmin, but figure it can save a step. Also the mapping isn't permanent (after the host is gone can we still map alias to hostname?)

Reply:
k8s has become the default for new installations and the hostname is not relevant there. We also have no idea what the hostname is here (or the pod name, etc).

Reply:
And yes, the string printed here is the same string you see in the first column of the `vtctldclient GetTablets` command output, and the reference for tablets that is used throughout Vitess.

Reply:
Ah okay, yes, this makes sense, thanks for the explanation. We will do that mapping to the pod name then after we receive the alias in the error

Reply:
I was wrong about the hostname not being available there. After more consideration, you convinced me. 🙂 I've added that context as well here: https://github.com/vitessio/vitess/pull/17558/commits/a136b86d897c94a98b65a00ebe7febb4ce00b5b1

Thanks for the good idea!